src/cms/utils/account_activation_utils.py

```diff
@@ -2,20 +2,7 @@
 This module contains helpers for the account activation process
 (also see :class:`~cms.views.authentication.account_activation_view.AccountActivationView`).
 """
-import logging
-
-from django.contrib import messages
 from django.contrib.auth.tokens import PasswordResetTokenGenerator
-from django.core.mail import send_mail, BadHeaderError
-from django.template.loader import render_to_string
-from django.utils.encoding import force_bytes
-from django.utils.http import urlsafe_base64_encode
-from django.utils.translation import ugettext as _
-
-from backend.settings import BASE_URL
-from ..models import Region
-
-logger = logging.getLogger(__name__)
 
 
 class AccountActivationTokenGenerator(PasswordResetTokenGenerator):
```

Comment:
If this file only contains the `AccountActivationTokenGenerator`, I would suggest renaming it to `account_activation_token_generator.py`.